package.json

```diff
@@ -10,9 +10,9 @@
     "type:check": "turbo run type:check",
     "type:emit": "turbo run type:emit",
     "build-storybook": "turbo run build-storybook",
-    "test": "turbo run test",
+    "test": "turbo run test -- --passWithNoTests",
```

Comment:
this flag was not necessary in the past and should not be necessary after this change

Reply:
When I tested a few packages failed because they had no tests.
But I have removed it as you suggested. If CI checks pass it is all good. 👍

Reply:
Update applied 👍